packages/terra-date-time-picker/src/DateTimePicker.jsx

```diff
@@ -14,6 +14,11 @@ import TimeClarification from './_TimeClarification';
 const cx = classNames.bind(styles);
 
 const propTypes = {
+  /**
+   * String that labels the current element. 'aria-label' must be present for accessibility.
+   */
+  ariaLabel: PropTypes.string,
```

Comment:
Description can be more detailed like the one you have added in date input and datePicker

Reply:
done https://github.com/cerner/terra-framework/pull/2031/commits/8d4603715fe10fee9e460c580b7f7c9635d09565